src/core/lib/gprpp/thd.h

```diff
@@ -119,12 +153,13 @@ class Thread {
   /// FAKE -- just a dummy placeholder Thread created by the default constructor
   /// ALIVE -- an actual thread of control exists associated with this thread
   /// STARTED -- the thread of control has been started
-  /// DONE -- the thread of control has completed and been joined
+  /// DONE -- the thread of control has completed and been joined/detached
```

Comment:
I think this comment is not accurate, correct? Detached threads never reach DONE state here.

Reply:
Removed, thanks. I think that is a side-effect of your fix in #18120 :)